variables.tf

```
@@ -68,6 +68,18 @@ variable "redis_timeout" {
   default     = "10m"
 }
 
+variable "use_redis_replicas" {
```

Using a common prefix helps to group similar/related attributes together in terraform module docs

```suggestion
variable "read_replicas_enabled" {
```